src/definitions.d.ts

```diff
@@ -44,6 +44,8 @@ export interface FilterConfig {
     sepia: number;
     useFont: boolean;
     fontFamily: string;
+    useFontSize: boolean;
```

Comment:
Making `font-size` the same for all elements on the page is a bad idea. All the browsers already have a nice Zoom feature, that scales everything proportionally.